snuba/datasets/entities/outcomes_raw.py

```diff
@@ -20,13 +21,14 @@
 class OutcomesRawEntity(Entity):
     def __init__(self) -> None:
         storage = get_storage(StorageKey.OUTCOMES_RAW)
-
+        read_columns = storage.get_schema().get_columns()
+        time_columns = ColumnSet([("time", DateTime())])
```

Comment:
Couldn't we just extended the original columns in the storage schema to add a time column? (I'm not sure if that would have other implications)

Reply:
The storage schema is supposed to reflect what is actually in the tables, while bucketed_time is a "magic" column that only exists on the Entity and not in the storage itself. That's why I added it only to the Entity.